.github/workflows/test.yml

```diff
@@ -190,6 +190,47 @@ jobs:
           pixi-url: https://github.com/prefix-dev/pixi/releases/download/v0.14.0/pixi-x86_64-unknown-linux-musl
       - run: pixi --version | grep -q "pixi 0.14.0"
 
+  pixi-url-bearer-token:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Move pixi.toml
+        run: mv test/old-pixi-lockfiles/* .
+      - name: Setup caddy
+        run: |
+          set -euo pipefail
+          curl -Ls -o caddy.tar.gz https://github.com/caddyserver/caddy/releases/download/v2.10.0/caddy_2.10.0_linux_amd64.tar.gz
+          tar -xzf caddy.tar.gz
+          chmod +x caddy
+
+          mkdir -p assets
+          curl -Ls -o assets/pixi https://github.com/prefix-dev/pixi/releases/download/v0.14.0/pixi-x86_64-unknown-linux-musl
+
+          cp .github/assets/pixi-url-auth-test/Caddyfile .
+
+          ./caddy run --config Caddyfile &
+      - run: ps -aux | grep caddy
+      - name: Run with wrong token
+        uses: ./
+        id: wrongtoken
+        with:
+          cache: false
+          pixi-url: http://localhost:8080/pixi
+          pixi-url-bearer-token: wrongtoken
+        continue-on-error: true
```

Comment:
didn't know this existed, very nice. then we can activate the commented out tests at the bottom of the file (i can do this in another pr)